### ISSUE TITLE
Improve Accessibility

### ISSUE THREAD

**Issue by Karvel** [Shift3/boilerplate-client-angular#166]
## Desired Use Case

The client should have improved accessibility. Certain semantic HTML elements should only appear once. The app should adjust focus to improve keyboard navigation.

## Actual/Current Behavior

These accessibility features are not yet implemented.

## Steps to Implement This Feature

  1. Ensure a top-level `<h1>` is available.
  2. Set the top-level `<h1>` to be viewable to screen readers only (assistive technology prefers having one relevant `<h1>` on each page). The designs for the app do not make an easy place to put a visible `<h1>` for all views.
  3. Convert the existing `<h1>` instances to `<h2>`
  4. Improve linking of labels to form elements in the dynamic form.
  5. Set up page focus for the routed components.
  6. Adjust some labels to improve accessibility.
